src/config/default-doc-config.js

```diff
@@ -0,0 +1,8 @@
+export const defaultDocConfig = {
+  publish: Boolean(process.env.CI),
+  entryPoint: 'src/index.ts',
```

Comment:
` isTypescript ? 'src/index.ts' : 'src/index.js',` became `'src/index.ts'`.

I'm not 100% sure if this option is used by the command. I'm pretty sure it's handled in someway through `forwardOptions`, but I am having trouble finding documentation on `ctx['--']`.

Reply:
Wanted to switch this since the coupling to utils comes with the side affect of requiring a package.json.

But, think I'll put it back as it was for backward compatibility and focus next on isolating that side effect.

Reply:
Rather than using a boolean condition to figure it out. Instead used a glob.

According to typedoc's documentation and passing tests, this is acceptable. https://typedoc.org/options/input/#entrypoint

Though, entrypoint is deprecated according to the documentation.